### ISSUE TITLE
process.runtime.go.cgo.calls metric should be Counter

### ISSUE THREAD

**Issue by vmihailenco** [open-telemetry/opentelemetry-go-contrib#2648]
`runtime.NumCgoCall()` returns a monotonic number which never goes down so I guess it should be a Counter.

https://github.com/open-telemetry/opentelemetry-go-contrib/commit/a5875205b0664b93cb05667e49c034b94576a4c7 changed process.runtime.go.cgo.calls from Counter to UpDownCounter without providing a reason.

**Comment by jmacd** [comment]
Agreed. I want to retire the library producing this metric, however, see https://github.com/open-telemetry/opentelemetry-go-contrib/pull/2643 and plan on discussing this topic in the 9/8 Go SIG call.
